webgpu.yml

```diff
@@ -697,8 +708,9 @@ enums:
       - name: render_pass_max_draw_count
         doc: |
           TODO
-      # TODO(#214): Move all of the surface sources into block 0x0001
       - name: surface_source_metal_layer
+        # TODO(#214): Move all of the surface sources into block 0x0001
+
```

Comment:
This doesn't seem like it should have changed, but otherwise LGTM!

Reply:
Oh, dunno how this came up here